recipe/meta.yaml

```
@@ -1,29 +1,29 @@
 {% set name = "hatch-fancy-pypi-readme" %}
-{% set version = "22.8.0" %}
+{% set version = "24.1.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hatch_fancy_pypi_readme-{{ version }}.tar.gz
-  sha256: da91282ca09601c18aded8e378daf8b578c70214866f0971156ee9bb9ce6c26a
+  sha256: 44dd239f1a779b9dcf8ebc9401a611fd7f7e3e14578dcf22c265dfaf7c1514b8
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<37]
   number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
   entry_points:
     - hatch-fancy-pypi-readme=hatch_fancy_pypi_readme.__main__:main
 
 requirements:
   host:
     - hatchling
     - pip
-    - python >=3.7
+    - python
   run:
     - hatchling
-    - python >=3.7
+    - python
     - tomli
     - typing-extensions
```

```suggestion
    - typing-extensions # [py<38]
```
https://github.com/hynek/hatch-fancy-pypi-readme/blob/24.1.0/pyproject.toml#L31